docker-compose.yml

```diff
@@ -141,7 +141,12 @@ services:
   build-rails-db:
     image: compliance-backend-rails
     entrypoint: ''
-    command: 'bundle exec rake db:create db:migrate'
+    command: sh -c '
+      bundle exec rake db:create db:migrate;
+      bundle exec rake db:test:prepare;
+      bundle exec rails db < cyndi_setup.sql;
```

Comment:
Is this setup targeted *only* for development? If not, can we move it to a migration?

Reply:
Yes, this is only for development. In other environments, we create the schema but don't create the view itself.